analysis_options.yaml

```diff
@@ -2,3 +2,7 @@ include: package:lints/recommended.yaml
 
 analyzer:
   exclude: [build/**]
+
+linter:
+  rules:
+    implementation_imports: false
```

Comment:
Why? I'd rather disable this on a case-by-case basis so we don't miss anything we _shouldn't_ be importing.

Reply:
@l7ssha enabled it [here](https://github.com/nyxx-discord/nyxx_extensions/commit/1f9506731b86fde51af22ee43ea61cf425950487#diff-9a967a7bb0393381dfdf9cce102a9cea39146829ab8b38e983d54ac8827486bc) and then removed it some time ago, I decided to re-enable it since I wasn't sure if we chose to disable with comments. (the analyzer is only complaining [here](https://github.com/nyxx-discord/nyxx_extensions/blob/4f0d5d2dca79084156e26df4e8a96a6ec763f113/lib/src/utils/emoji.dart#L5-L6))

Reply:
I feel like we should disable with comments (what I meant by "case-by-case").